frontend/packages/ceph-storage-plugin/src/constants/index.ts

```diff
@@ -15,6 +15,7 @@ export const OCS_INTERNAL_CR_NAME = 'ocs-storagecluster';
 export const NO_PROVISIONER = 'kubernetes.io/no-provisioner';
 export const OCS_SUPPORT_ANNOTATION = 'features.ocs.openshift.io/enabled';
 export const OCS_DEVICE_SET_REPLICA = 3;
+export const OCS_DEVICE_SET_ARBITER_REPLICA = 4;
```

Comment:
```suggestion
export const OCS_DEVICE_SET_ARBITER_REPLICA = OCS_DEVICE_SET_REPLICA + 1;
```
Is this how we ended up with 4?

Reply:
2 nodes * 2 zones. Let's not do +1 as it's different from what we do normally.

Reply:
oh okay, let it be as it is in that case.